src/pybind/mgr/dashboard/controllers/telemetry.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from . import ApiController, RESTController
+from .. import mgr
+from ..exceptions import DashboardException
+from ..security import Scope
+
+
+@ApiController('/telemetry', Scope.CONFIG_OPT)
+class Telemetry(RESTController):
+
+    @RESTController.Collection('GET')
+    def report(self):
+        """
+        Get Ceph and device report data
+        :return: Ceph and device report data
+        :rtype: dict
+        """
+        return mgr.remote('telemetry', 'get_report', 'all')
+
+    def bulk_set(self, enable=True, license_name=None):
```

Comment:
mmm... `bulk_set`? Ok ok, with a regular `set` this doesn't work because it tries to fetch some resource ID or something like that? No big problem, singleton resources are corner cases, but if some reviewer stumbles upon this line they will scratch their heads as `bulk_set` sounds like something unrelated :thinking:

Reply:
Yes, I agree with you. I'm not happy with the `bulk_set` neither. Yes, I had the issues with the resource ID when using `set` the endpoint e.g. looked like `/telemetry/enable` - `{"enabled": true, "license": ...}` and we wanted to use `/telemetry` - `{"enabled": true, "license": ...}`.
Do you have a better idea? - Otherwise I can also add a comment about it to the code.

Reply:
In controllers/__init__.py you may add another alias for bulk_set, like singleton_set (as this is a singleton resource, hence no IDs are needed)?